packages/plugins/users-permissions/server/controllers/validation/user.js

```diff
@@ -6,24 +6,21 @@ const deleteRoleSchema = yup.object().shape({
   role: yup.strapiID().required(),
 });
 
-const createUserBodySchema = yup
-  .object()
-  .shape({
-    email: yup
-      .string()
-      .email()
-      .required(),
-    username: yup
-      .string()
-      .min(1)
-      .required(),
-    password: yup
-      .string()
-      .min(1)
-      .required(),
-    role: yup.strapiID(),
-  })
-  .noUnknown();
+const createUserBodySchema = yup.object().shape({
```

Comment:
Why not add the fields instead of allowing extra ones ?

Reply:
Because the model can be extended by the user and it would break here then

Reply:
Totally ^^